### PR TITLE
Delete Logs Older than X Seconds

### PR DESCRIPTION
**Pull request by christyjacob4**
Function to delete audit logs older than X seconds

### CODE REVIEW

File: src/Audit/Adapters/MySQL.php [hunk 1]
```diff
@@ -116,6 +116,20 @@ public function getLogsByUserAndActions(string $userId, array $actions):array
...
+     * Delete logs older than $days Days 
+     * 
+     * @param int $days  
+     */
+    public function deleteLogsOlderThan(int $days)
+    {
+        $st = $this->getPDO()->prepare('SELECT *
+        FROM `'.$this->getNamespace().'.audit.audit`
+            WHERE DATEDIFF(day, `time`, getDate()) > :days');
+        $st->bindValue(':days', $days, PDO::PARAM_STR);
```
Comment (eldadfux) [inline src/Audit/Adapters/MySQL.php:129]: Replace with PDO::PARAM_STR with PDO::PARAM_INT

**Comment by eldadfux** [comment]
You got to ways to test DB operation:
1. Use the CI - commit your changes and view the test outcome on the Travis UI.
2. Run a local MariaDB/MySQL Docker container

File: src/Audit/Adapters/MySQL.php (hunk 1 above)
Comment (christyjacob4) [reply src/Audit/Adapters/MySQL.php]: Done

**Comment by christyjacob4** [comment]
Ready 👍

**Comment by eldadfux** [comment]
🎈

**Pull request merged**